actors/builtin/miner/miner_actor.go

```diff
@@ -187,6 +187,7 @@ type ChangePeerIDParams struct {
 }
 
 func (a Actor) ChangePeerID(rt Runtime, params *ChangePeerIDParams) *adt.EmptyValue {
+	// TODO: Max size of peer ID?
```

Comment:
?

Reply:
If this is still TODO when you merge, please reference the issue in the comment.

Reply:
Will do. I was going to try to resolve it now, but I'm not even sure if limits make sense here.